src/components/cards/TappableCard.tsx

```diff
@@ -24,15 +24,19 @@ const TappableCardComponent = ({ children, disabled = false, onPress, ...cardPro
   const theme = useTheme()
   const styles = getStyles(theme)
 
+  const renderTouchableChildren = (
```

Comment:
This should be a function as all `render*` functions are.

Reply:
Doesn't work here, TS complains about not allowing functions as children